package.xml

```diff
@@ -60,14 +56,12 @@
   <depend>genmsg</depend>
   <depend>genpy</depend>
   <depend>geometry_msgs</depend>
-  <depend>glut</depend>
+  <depend>glut</depend> <!-- ??? -->
   <depend>gps_common</depend>
-  <depend>grid_map_cv</depend>
   <depend>grid_map_msgs</depend>
   <depend>grid_map_ros</depend>
   <depend>grid_map_visualization</depend>
-  <depend>gscam</depend>
-  <depend>gstreamer0.10-plugins-good</depend>
+  <depend>gscam</depend> Not supported in noetic -->
```

Comment:
This comment looks like a syntax error. Is there a leading `<!--` somewhere I'm not seeing?

Reply:
Yes, an editing blunder. Fixed.